src/main/java/Duke.java

```diff
@@ -1,10 +1,126 @@
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+
 public class Duke {
+
+    public static String indent = "     ";
+
+    // init array of Task objects
+    public static ArrayList<Task> allTasks = new ArrayList<>();
+
+    // helper function to convert string array to string
+    // not just string rep like what Arrays.toString(strArr) does
+    public static String convertToString(String[] strArr) {
+//        StringBuilder builder = new StringBuilder();
+//        for(String s : strArr) {
+//            builder.append(s + " ");
+//        }
+//        return builder.toString();
```

Comment:
Perhaps these can be extracted out if not in use? You can consider storing these comments somewhere accessible if you want to refer to it in future, but I think it would be better to extract them out now to make the commits in your pull request more readable!